openapi.yml

```diff
@@ -4941,6 +4977,54 @@ components:
                         colour: RED
                         last_name: some text
                         first_name: some text
+        NotificationsCreateParams:
+            description: ''
+            required:
+                - target
+                - message
+                - channels
+            type: object
+            properties:
+                target:
+                    description: Creates a notification for signins
+                    type: string
+                message:
+                    description: ''
+                    type: string
+                signin_filters:
+                    $ref: '#/components/schemas/NotifcationsSigninFilterParams'
+                    description: ''
+                channels:
+                    description: ''
+                    type: array
+                    items:
+                        type: string
+        NotifcationsSigninFilterParams:
+            description: ''
+            required: []
+            type: object
+            properties:
+                location_ids:
+                    description: ''
+                    type: array
+                    items:
+                        type: integer
+                signins_ids:
+                    description: ''
+                    type: array
+                    items:
+                        type: integer
+                with_signed_in:
```

Comment:
I didn't understand the properties "with_signed_in", "signin_before" and "signin_after". Are they something to make the endpoint future proof or something needed for the current feature?

Reply:
This is to allow to send notifications to visitors who signed in a certain date range. For example, if a company discovers someone was sick while the visitor was visiting the company, they want to send them a notification asking them to get themselves checked.

Reply:
interesting... so I guess it is more like a future proof spec in case we need this kind of date filter. Good job!